.golangci.yml

```diff
@@ -0,0 +1,115 @@
+run:
```

Comment:
Added a `golangci-lint` config other contributors could use to ensure some quality.